doc/mkdocs/mkdocs.yml

```diff
@@ -150,15 +150,16 @@ nav:
         - 'operator value_t': api/basic_json/operator_value_t.md
         - 'operator[]': api/basic_json/operator[].md
         - 'operator=': api/basic_json/operator=.md
+        - 'operator+=': api/basic_json/operator+=.md
         - 'operator==': api/basic_json/operator_eq.md
         - 'operator!=': api/basic_json/operator_ne.md
         - 'operator<': api/basic_json/operator_lt.md
```

Comment:
I think we should either have

- `<`, `>`, `<=`, `>=` (like cppreference) or
- `<`, `<=`, `>`, `>=` (similar to the order before).

But

- `<`, `<=`, `>=`, `>`

feels odd to me.

Reply:
Hm.
less than, less than or equal, (equal), greater than or equal, greater than
Feels natural to me but I guess this is going to be subjective either way. Might as well pick the cppreference order then.

Reply:
Yes, pleae.